deliverystream/src/test/java/com/amazonaws/kinesisfirehose/deliverystream/CreateHandlerTest.java

```diff
@@ -294,6 +294,56 @@ public void testCreateDeliveryStreamWithSplunkConfiguration() {
         verify(proxy, times(2)).injectCredentialsAndInvokeV2(any(DescribeDeliveryStreamRequest.class), any());
     }
 
+    @Test
+    public void testCreateDeliveryStreamWithHttpEndpointConfiguration() {
+        final ResourceModel model = ResourceModel.builder()
```

Comment:
Does it makes sense to use `lombok.val` instead (`val varName` instead of `final Type varName`) for brevity. Not sure if we follow some convention around that.